<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OTPNitro GUI</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/main_icon_32.png</normaloff>:/icons/main_icon_32.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPlainTextEdit" name="msgBox">
      <property name="font">
       <font>
        <family>Monospace</family>
        <weight>75</weight>
        <bold>true</bold>
        <strikeout>false</strikeout>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>572</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionClear"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuCrypto">
    <property name="title">
     <string>Crypto</string>
    </property>
    <addaction name="actionCrypt"/>
    <addaction name="actionDecrypt"/>
    <addaction name="separator"/>
    <addaction name="actionEncode"/>
   </widget>
   <widget class="QMenu" name="menuBooks">
    <property name="title">
     <string>Books</string>
    </property>
    <addaction name="actionRefresh_books"/>
    <addaction name="actionGenerate_book"/>
    <addaction name="separator"/>
    <addaction name="actionBurn_page"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCrypto"/>
   <addaction name="menuBooks"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/clear.png</normaloff>:/icons/clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/open.png</normaloff>:/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/config.png</normaloff>:/icons/config.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/exit.png</normaloff>:/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCrypt">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/crypt.png</normaloff>:/icons/crypt.png</iconset>
   </property>
   <property name="text">
    <string>Crypt</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionDecrypt">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/decrypt.png</normaloff>:/icons/decrypt.png</iconset>
   </property>
   <property name="text">
    <string>Decrypt</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionEncode">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/encode.png</normaloff>:/icons/encode.png</iconset>
   </property>
   <property name="text">
    <string>Encode</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionRefresh_books">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/refresh.png</normaloff>:/icons/refresh.png</iconset>
   </property>
   <property name="text">
    <string>Refresh books</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionGenerate_book">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/generate.png</normaloff>:/icons/generate.png</iconset>
   </property>
   <property name="text">
    <string>Generate book</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionBurn_page">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/burn.png</normaloff>:/icons/burn.png</iconset>
   </property>
   <property name="text">
    <string>Burn page</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/help.png</normaloff>:/icons/help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/about.png</normaloff>:/icons/about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
